docs/guides/11-add-widgets.md

```diff
@@ -1,8 +1,8 @@
 ## Add widgets
 
-**Widgets** often provide additional information about a dataset that is not symbolized on a map to facilitate understanding and exploration. On an election map, you can style the map by the winner, but also provide additional information from the data, for example, how many total votes did each party receive? Or, how many people voted?
+[**Widgets**](https://carto.com/help/glossary/#widget) provide additional information about a dataset that is not symbolized on a map to facilitate understanding and exploration. On an election map, you can style the map by the winner, but also provide additional information from the data, for example, how many total votes did each party receive? Or, how many people voted?
 
-The first thing you need to know to create widgets is the concept of **variables**. A variable is a name in a CARTO VL visualization that has an expression bound to it. It's simply a way to name expressions to be referenced later. For example, let's say there was a magic function that returned our widget information like `myWidgetExpression()`, how do we actually get that information?
+The first thing you need to know to create widgets is the concept of [**variables**](https://carto.com/help/glossary/#variable). A variable is a name in a CARTO VL visualization that has an expression bound to it. It's simply a way to name expressions to be referenced later. For example, let's say there was a magic function that returned our widget information like `myWidgetExpression()`, how do we actually get that information?
```

Comment:
Same here. In fact, I'm surprised there is a variable term in the glossary talking just about CARTO VL

Reply:
yep! that's what i realized about a lot of the glossary terms... they are actually for CARTO VL!